storage: TestMVCCDeleteRangeConcurrentTxn failed

storage.TestMVCCDeleteRangeConcurrentTxn [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8004682?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8004682?buildTab=artifacts#/) on master @ [7c8e7012b73a598f3aa1da9bbf422e833706f5d3](https://github.com/cockroachdb/cockroach/commits/7c8e7012b73a598f3aa1da9bbf422e833706f5d3):


```
=== RUN   TestMVCCDeleteRangeConcurrentTxn
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/c5eb8fc8b8e683c19f3c3e4238f64094/logTestMVCCDeleteRangeConcurrentTxn4218633289
    test_log_scope.go:79: use -show-logs to present logs inline
    mvcc_test.go:1627: -- test log scope end --
    mvcc_test.go:1627: Leaked goroutine: goroutine 20 [chan receive]:
        github.com/cockroachdb/cockroach/pkg/util/log.flushDaemon()
        	github.com/cockroachdb/cockroach/pkg/util/log/log_flush.go:75 +0x5b
        created by github.com/cockroachdb/cockroach/pkg/util/log.init.5
        	github.com/cockroachdb/cockroach/pkg/util/log/log_flush.go:41 +0x25
--- FAIL: TestMVCCDeleteRangeConcurrentTxn (5.04s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/storage
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMVCCDeleteRangeConcurrentTxn.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/util/leaktest/leaktest.go
+++ b/pkg/util/leaktest/leaktest.go
@@ -58,6 +58,10 @@ func interestingGoroutines() map[int64]string {
 			strings.Contains(stack, "sentry-go.(*HTTPTransport).worker") ||
 			// Seems to be gccgo specific.
 			(runtime.Compiler == "gccgo" && strings.Contains(stack, "testing.T.Parallel")) ||
+			// Ignore intentionally long-running logging goroutines that live for the
+			// duration of the process.
+			strings.Contains(stack, "log.flushDaemon") ||
+			strings.Contains(stack, "log.signalFlusher") ||
 			// Below are the stacks ignored by the upstream leaktest code.
 			strings.Contains(stack, "testing.Main(") ||
 			strings.Contains(stack, "testing.tRunner(") ||

